src/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -0,0 +1,618 @@
+            cChange++;
+        if (strCleanup(addrNew.MailingState__c) != strCleanup(addrOld.MailingState__c) && addrOld.MailingState__c != null)
+            cChange++;
+        if (strCleanup(addrNew.MailingPostalCode__c) != strCleanup(addrOld.MailingPostalCode__c) && addrOld.MailingPostalCode__c != null)
+            cChange++;
+        if (strCleanup(addrNew.MailingCountry__c) != strCleanup(addrOld.MailingCountry__c) && addrOld.MailingCountry__c != null)
+            cChange++;
+        if (includeAddressType && strCleanup(addrNew.Address_Type__c) != strCleanup(addrOld.Address_Type__c) && addrOld.Address_Type__c != null)
+            cChange++;
+        return cChange <= 1;
+    }
+    
+    /*******************************************************************************************************
+    * @description utility to remove all whitespace and to lowercase the string
+    * @param str the string to clean up
+    * @return string
+    */ 
+    private static string strCleanup(string str) {
+        if (str != null) return str.replace(' ', '').toLowerCase();
+        else return '';
+    }
+    
+    /*******************************************************************************************************
+    * @description Utility to copy Address fields between Address objects.
+    * @param addrSrc the source Address
+    * @param addrSrc the destination Address
+    * @return void
+    */ 
+    public static void copyAddressObj(Address__c addrSrc, Address__c addrDst) {
+        addrDst.MailingStreet__c = addrSrc.MailingStreet__c;
+        addrDst.MailingStreet2__c = addrSrc.MailingStreet2__c;
+        addrDst.MailingCity__c = addrSrc.MailingCity__c;
+        addrDst.MailingState__c = addrSrc.MailingState__c;
+        addrDst.MailingPostalCode__c = addrSrc.MailingPostalCode__c;
+        addrDst.MailingCountry__c = addrSrc.MailingCountry__c;
+        addrDst.Geolocation__Latitude__s = addrSrc.Geolocation__Latitude__s;
+        addrDst.Geolocation__Longitude__s = addrSrc.Geolocation__Longitude__s;
+        if (addrSrc.Address_Type__c != null)
+            addrDst.Address_Type__c = addrSrc.Address_Type__c;
+    }
+    
+     /*******************************************************************************************************
+    * @description Utility to copy Address fields between Contacts and Accounts (or vice-versa).
+    * Handles instances where State and Country picklists are enabled, and multiline street addresses.
+    * @param sobjSrc the source Contact or Account
+    * @param strFieldPrefixSrc the address fields to copy from, ie., Mailing, Other, Shipping, Billing
+    * @param sobjDst the destination Contact or Account
+    * @param strFieldPrefixDst the address fields to copy to, ie., Mailing, Other, Shipping, Billing
+    * @return void
+    */ 
+    public static void copyAddressStdSObj(SObject sobjSrc, string strFieldPrefixSrc, SObject sobjDst, string strFieldPrefixDst) {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.